ambry-store/src/test/java/com.github.ambry.store/PersistentIndexTest.java

```diff
@@ -712,7 +712,8 @@ public void testMissingEntries()
   }
 
   @Test
-  public void testRollingIndex() {
+  public void testRollingIndex()
+      throws IOException {
     MockClusterMap map = null;
```

Comment:
this is in a different test altogether which I had to change just for the signature. I think at this time, it is better to not go down the route of changing these variable names.